### ISSUE TITLE
[a11y] Unable to open `Main menu` when `Talkback` is ON

### ISSUE THREAD

**Issue by hffvld** [brave/brave-browser#38807]
### Description

Unable to open `Main menu` when `Talkback` is ON.

### Steps to reproduce

1. Go to the device settings > Accessibility > Activate `Talkback`
2. Launch Brave
3. Double tap the `three-dot` to open the `Main menu` > *Observe*

### Actual result

Unable to open `Main menu` when `Talkback` is ON.

https://github.com/brave/brave-browser/assets/128532965/9788877b-b5b8-4aaa-82e6-65c3c0df6b8d

### Expected result

The `Main menu` must be opened on **double tap** when `Talkback` is ON.

### Reproduces how often

Easily reproduced

### Brave version

Brave build: 1.67.105
Chromium: 126.0.6478.26 (Official Build) beta (64-bit)

### Device

- Brand/Model: Pixel 7
- Android version: panther_beta-user 14 AP31.240426.023.B4 release-keys

### Channel information

- [X] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc @pet
